content_scripts/CameraVis.js

```diff
@@ -700,6 +702,26 @@ void main() {
             this.startWebRTC();
             this.restorePatches();
         }
+        
+        onAnimationFrame() {
+            let now = performance.now();
+            for (let camera of Object.values(this.cameras)) {
+                if (camera.mesh.__hidden) {
+                    camera.mesh.visible = false;
+                    continue;
+                }
+                if (now - camera.lastUpdate > 5000) {
+                    camera.remove();
+                    delete this.cameras[camera.id];
+                    this.callbacks.onCameraVisRemoved.forEach(cb => {
+                        cb(camera);
+                    });
+                } else if (!camera.mesh.visible) {
+                    camera.mesh.visible = true;
+                }
+            }
+            window.requestAnimationFrame(() => this.onAnimationFrame());
```

Comment:
This will increase memory pressure by allocating a new closure every frame. It would be better to do
```javascript
this.onAnimationFrame = this.onAnimationFrame.bind(this);
window.requestAnimationFrame(this.onAnimationFrame);
```

Reply:
(in the constructor)